man/tpm2_import.1.md

```diff
@@ -21,30 +21,31 @@ OPTIONS
 
 These options control the key importation process:
 
-  * `-G`, **--import-key-alg**=_ALGORITHM_:
+  * **-G**, **--import-key-alg**=_ALGORITHM_:
     The hash algorithm to use. Algorithms should follow the
```

Comment:
Not form your patch, but I think this description is wrong. -G should be the Algorithm for the key iteself, -g is the name algorithm which is a hashing algorithm.

Reply:
had to dig through code and docs to confirm, but I believe you're right. I'll fix and resubmit.